src/components/Education.jsx

```diff
@@ -22,42 +24,73 @@ const FeatureCard = ({
   content1,
   content2,
   index,
-}) => (
-  <div
-    className={`flex flex-row p-6 rounded-[20px]
-	${index === educationList.length - 1 ? "mb-0" : "mb-6"} feature-card`}
-  >
+}) => {
+  const { isDarkMode } = useTheme();
+  return (
     <div
-      className={`w-[64px] h-[64px] rounded-full ${styles.flexCenter} bg-dimBlue`}
+      className={`flex flex-row p-6 rounded-[20px] ${
+        index === educationList.length - 1 ? "mb-0" : "mb-6"
+      } ${isDarkMode ? "feature-card" : "bg-gray-100"}`}
```

Comment:
Lets not have direct classes for the light mode. Keep the similar behavior in terms of coding style for light mode as well.